src/components/charts/timebank-piechart.tsx

```diff
@@ -40,8 +40,16 @@ const TimebankPieChart = ({ personDailyEntry }: Props) => {
           cx="50%"
           cy="50%"
           outerRadius={50}
-          label={renderCustomizedLabel}
-        >
+          label={(entry) => {
+            if (!entry.dataKey) {
+              return null; // do not render any label if dataKey is 0 or empty
```

Comment:
Although sometimes its good to leave comments in the code if what is happening is unclear, I would say in this case these comments are not needed and can be removed.